.codegen/api.go.tmpl

```diff
@@ -162,8 +162,11 @@ func (a *{{.Service.Name}}API) {{.PascalName}}AndWait(ctx context.Context{{if .R
 {{.Comment "// " 80}}
 // 
 // This method is generated by Databricks SDK Code Generator.
-func (a *{{.Service.Name}}API) {{.PascalName}}All(ctx context.Context{{if .Request}}, request {{.Request.PascalName}}{{end}}) ([]{{.Pagination.Entity.PascalName}}, error) {
+func (a *{{.Service.Name}}API) {{.PascalName}}All(ctx context.Context{{if .ListingRequest}}, listing {{.ListingRequest.PascalName}}{{end}}) ([]{{.Pagination.Entity.PascalName}}, error) {
```

Comment:
you're incorrect, there's rename happening from the existing request to new request. Without renames, we have a breaking change for customer code.

<img width="584" alt="image" src="https://github.com/databricks/databricks-sdk-go/assets/259697/5bcafbe3-03ac-45da-bb98-7adbf4f174e4">

Reply:
also - https://github.com/databricks/databricks-sdk-go/pull/569#discussion_r1290116601

Reply:
okay, renamed and introduced `{{.RealRequest}}`.

Reply:
I misunderstood and the new name is only exposed in the lower-level interface, which is more "internal" than the actual API interface. Seems OK

Reply:
I think this is good though since it should be easier for developers on OpenAPI generator to understand, with the idea that `Request` and `Response` correspond to the user-facing request and response, and `RealRequest` is an implementation detail.